lib/legacy.coffee

```diff
@@ -17,8 +17,17 @@ license = require './license'
 asSlug = require('./page').asSlug
 newPage = require('./page').newPage
 
+preLoadCustomEditors = (catalog) ->
+  catalog
+    .filter((entry) -> entry.customEditor)
```

Comment:
as implemented here, the plugin authors include this in `factory.json`
```
  ...
  "customEditor": true,
  ...
```

Reply:
For example, here's slackmatic plugin's factory.json
https://github.com/dobbs/wiki-plugin-slackmatic/blob/master/factory.json

Reply:
I like this solution. It offers a nice compromise between built-in plugins and load-on-first-use plugins, the two choices we have today. The server admin is choosing to selectively build-in a capability for all users of that farm only.

```
{
  "name": "Slackmatic",
  "title": "Slackmatic Plugin",
  "category": "other",
  "customEditor": true
}
```
The example I gave was a plugin's about page talking to itself so the choice of terms was of little consequence. Here a plugin is talking to the Factory about Factory issues where everything is custom. Maybe "custom" is redundant or "true" is missing an opportunity to be more precise. I don't have a specific suggestion. Just recognizing the uniqueness of the channel.